src/NServiceBus.AcceptanceTests/ApprovalFiles/When_pipelines_are_built.Should_preserve_order.approved.txt

```diff
@@ -0,0 +1,73 @@
+context0 => GetBehavior(context0, 0).Invoke(context0, value(System.Func`2[NServiceBus.Pipeline.IOutgoingPublishContext,System.Threading.Tasks.Task])),
```

Comment:
the behaviors can't really be identified by this anymore. Would this even detect swapped behaviors?

Reply:
Yep that is one of the downsides of this change. Since the pretty print method is owned by us I could give it a go and see how I can tweak it.

Reply:
> Yep that is one of the downsides of this change. Since the pretty print method is owned by us I could give it a go and see how I can tweak it.

that would be 👨‍🍳👌